src/Symfony/Component/Validator/Tests/Constraints/CardSchemeValidatorTest.php

```diff
@@ -102,6 +102,12 @@ public function getValidNumbers()
             array('MAESTRO', '6594371785970435599'),
             array('MASTERCARD', '5555555555554444'),
             array('MASTERCARD', '5105105105105100'),
+            array('MASTERCARD', '2221005555554444'),
+            array('MASTERCARD', '2230000000000000'),
+            array('MASTERCARD', '2300000000000000'),
```

Comment:
The reason for testing `23xxx` and `26xxx` is beceause they are both in the range where only the 2 first digits need checking.
Therefor i feel that testing `24xxx` and `25xxx` is redundant, but i can add them if needed ?